daffodil-cli/bin.LICENSE

```diff
@@ -682,6 +682,404 @@ is subject to the terms and conditions of the following licenses.
     FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
     DEALINGS IN THE SOFTWARE.
 
+- com.siemens.ct.exi.exificient-<VERSION>.jar
+- com.siemens.ct.exi.exificient-core-<VERSION>.jar
+- com.siemens.ct.exi.exificient-grammars-<VERSION>.jar
+  This product bundles 'Exificient' from the above files.
+  These files are available under the MIT license:
+
+    The MIT License (MIT)
+
+    Copyright (c) 2007-2018 Siemens AG
+
+    Permission is hereby granted, free of charge, to any person obtaining a copy
+    of this software and associated documentation files (the "Software"), to deal
+    in the Software without restriction, including without limitation the rights
+    to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+    copies of the Software, and to permit persons to whom the Software is
+    furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included in
+    all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+    THE SOFTWARE.
+
+- com.sun.xml.bind.jaxb-core-<VERSION>.jar
+- com.sun.xml.bind.jaxb-impl-<VERSION>.jar
+- javax.activation.activation-<VERSION>.jar
+- javax.xml.bind.jaxb-api-<VERSION>.jar
+  This product bundles 'JAXB' and 'Activation Framework' from the above files.
+  These files are available under the CDDL license:
+
+    COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Version 1.1
```

Comment:
CDDL is one of those "weak copyleft" licenses where ASF requires it be "appropriately labeled":

https://www.apache.org/legal/resolved.html#weak-copyleft-licenses

We did this for Saxon (which is MPL 2.0 and also requires this) by adding a blurb in the `daffodil-cli/README.md` file. We should probably do the same for these JAXP dependencies.

Reply:
Done, ready for second reviewer.